### DISCUSSION TITLE
Option for chevron shown in example?

### DISCUSSION THREAD

**Discussion by MillionIncredible** [glennflanagan/react-collapsible#245] [Q&A]
Is there an option for the chevron shown in the example?

**Comment by MillionIncredible** [comment]
Ah nevermind, it's done in css with :after, right?

**Comment by glennflanagan** [comment]
The chevron is just applied using CSS in the example https://github.com/glennflanagan/react-collapsible/blob/develop/example/src/sass/components/_Collapsible.scss#L36 so you could use the same selectors to apply your own icon.

**Reply by MillionIncredible** [reply]
Thanks mate!
